index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="/testing/portofolio/style.css">
+    <link rel="stylesheet" href="/testing/portofolio/mediaqueries.css">
+    <link rel="stylesheet" href="/testing/portofolio/script.js">
```

Comment:
De ce ai toate URL-urile prefixate cu `/testing/portofolio`?

Vreau să îmi arăți unde faci deploy și dacă funcționează.

Reply:
Mulțumesc pentru review-uri, am sesizat și eu eroarea acum când am încercat sa l încarc pe netlify. Văzusem ca nu vede stylesheet-urile din cauza ca atunci când m ai ajutat cu încărcarea proiectul pe git hub, apăsasem eu greșit, și în loc sa l cloneze, mi s a creat automat un alt folder "Git-hub" în care s a pus tot proiectul. Asa ca va trebui sa fac din nou deploy ca sa corespundă url urile. Lucrez acum la asta